brands/BOM/tokens/index.js

```diff
@@ -434,6 +434,25 @@ export default {
 		},
 	},
 
+	progressBar: {
+		borderWidth: '1px',
```

Comment:
When we go to rems, don't change the borders to rems. They should always be 1px.